test/long/org/apache/cassandra/dht/tokenallocator/NoReplicationTokenAllocatorTest.java

```diff
@@ -166,11 +200,15 @@ private void grow(NoReplicationTokenAllocator<Unit> t, int targetClusterSize, To
             if (verifyMetrics)
             {
                 updateSummary(t, su, st, true);
-                double maxExpected = 1.0 + tc.spreadExpectation() * strategy.spreadExpectation() / perUnitCount;
+                double maxExpected = 1.0 + tc.spreadExpectation() * strategy.spreadExpectation() / perUnitCount / Math.pow(skewFactor, 2);
```

Comment:
the threshold here has to be relaxed when dealing with skewed clusters, because the tests assert that skew is below this threshold for all intermediate states, which won't always be the case.